src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+      class="searchForm"
+    >
+      <label for="bigSearchBar">
+        <img
+          src="images/Search.svg"
+          alt="searchIcon"
+          class="bigSearchBarIcon"
+        >
+        <input
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="bigSearchBar"
```

Comment:
the same, [checklist p.2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)